src/analysis/retail/warlock/affliction/constants.ts

```diff
@@ -1,29 +1,33 @@
 import SPELLS from 'common/SPELLS';
+import covenantSpells from 'common/SPELLS/shadowlands/covenants/warlock'
+import talents from 'common/TALENTS/warlock';
 import Combatant from 'parser/core/Combatant';
 
 const defaultDurations = {
+  // Baseline spells
   [SPELLS.AGONY.id]: 18000,
   [SPELLS.CORRUPTION_DEBUFF.id]: 14000,
-  [SPELLS.SIPHON_LIFE_TALENT.id]: 15000,
-  [SPELLS.UNSTABLE_AFFLICTION.id]: 16000,
   // Talents
-  [SPELLS.PHANTOM_SINGULARITY_TALENT.id]: 16000,
-  [SPELLS.VILE_TAINT_TALENT.id]: 10000,
-  // Covenant dots
-  [SPELLS.SCOURING_TITHE.id]: 18000,
-  [SPELLS.SOUL_ROT.id]: 8000,
-  [SPELLS.IMPENDING_CATASTROPHE_DEBUFF.id]: 12000,
+  [talents.SIPHON_LIFE_TALENT.id]: 15000,
+  [talents.UNSTABLE_AFFLICTION_TALENT.id]: 16000,
+  [talents.PHANTOM_SINGULARITY_TALENT.id]: 16000,
+  [talents.VILE_TAINT_TALENT.id]: 10000,
+  [talents.SOUL_ROT_TALENT.id]: 8000,
+  // Shadowlands Covenant Abilities
+  [covenantSpells.SCOURING_TITHE.id]: 18000,
+  [covenantSpells.SOUL_ROT.id]: 8000,
+  [covenantSpells.IMPENDING_CATASTROPHE_DEBUFF.id]: 12000,
```

Comment:
you don't have to, but commenting that it *is* okay to delete these if they're not present in the talent trees

Reply:
Sounds good! Yeah I think I'm going to have multiple PRs bringing this spec in line with some others, so I'll tackle a cleanup of SL abilities then, if that's ok?